doc/nrf/snippets/nrf91_modem_trace_uart.rst

```diff
@@ -0,0 +1,28 @@
+.. _nrf91_modem_trace_uart_snippet:
+
+nRF91 modem tracing with UART backend
+#####################################
+
+The snippet is tailored for tracing on the nRF91-DKs but can work with other applications as well.
```

Comment:
This sentence is a bit confusing, do you mean applications or boards?

Reply:
boards*